SwiftUIProject/SwiftUIProject/Helpers/String+Extension.swift

```diff
@@ -0,0 +1,51 @@
+// String+Extension.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Extension String for format text in phone number
+extension String {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let numbersRange = "[^0-9]"
+        static let emptyString = ""
+        static let emptyStringWithSpace = " "
+        static let xElement: Character = "X"
+        static let numberCodeWithPlus = "+7"
+        static let numberCode: Character = "7"
```

Comment:
```suggestion
        static let xElement  = "X"
        static let numberCodeWithPlus = "+7"
        static let numberCode = "7"
```
Мне ставили коммент чтобы я не указывал явно типы данных в константах

Reply:
Оставлю открытым, хочу узнать, что скажут менторы
это мне потом в коде каждый раз делать приведение типов?